### DISCUSSION TITLE
Metadata not updating on generated images past the first one

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#1703] [General]
I'd noticed this awhile back but assumed it was a something that would be quickly fixed.

After generating more than one image on the T2A tab, the metadata doesn't update when you select different images in the results. That is, the seed value stays frozen on the first seed that was used and doesn't show the seed for any other subsequent image. This also affects dynamic prompting (if you have it turned on, not required) in that the first image metadata is always displayed on subsequent images which have different prompt words.

**Comment by vladmandic** [comment]
i don't know what is T2A tab

**Comment by afbagwell** [comment]
Sorry I meant T2I ... text to image.
